warehouse/templates/manage/project/history.html

```diff
@@ -169,8 +169,31 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
     </small>
 
     {# Display API token events #}
+    {% elif event.tag == EventTag.Project.ShortLivedAPITokenAdded %}
+    <strong>{% trans %}Short-lived API token created{% endtrans %}</strong>
+    <small>{% trans %}Permissions: Can upload to this project{% endtrans %}</small>
+    {% if event.additional.expires %}
+    <small>
+      {% trans %}Expiration: {% endtrans %} {{ humanize(event.additional.expires | ctime, relative="false", time="true") }}
+    </small>
+    <small>
+      {% trans %} Expiration Status: {% endtrans %}
+      {% if (event.additional.expires | ctime) > now() %}
+        {% trans %}Active{% endtrans %}
+      {% else %}
+        {% trans %}Expired{% endtrans %}
+      {% endif %}
+    </small>
+    {% endif %}
+    <small>
+      {% trans %}Creator: {% endtrans %}
+      {{ event.additional.provider_name }}:
+      <a href="{{ event.additional.provider_url }}">
+        {{ event.additional.provider_spec }}
```

Comment:
I think this should just be `provider_url` (and there's at least one other place we're showing the spec as well)

Reply:
Yeah, I believe we show it in the registered providers table. I'll update there as well.

Reply:
(I'll do that after I'm merged on #12479, since it'll need deconflicting.)

Reply:
Merged!

Reply:
Updated and tweaked the table to contain the provider URL instead.